spec/models/service_template_orchestration_spec.rb

```diff
@@ -100,4 +100,52 @@
       end
     end
   end
+
+  describe '#create_catalog_item' do
+    let(:ra1) { FactoryGirl.create(:resource_action, :action => 'Provision') }
+    let(:ra2) { FactoryGirl.create(:resource_action, :action => 'Retirement') }
+    let(:service_dialog) { FactoryGirl.create(:dialog) }
+    let(:template) { FactoryGirl.create(:orchestration_template) }
+    let(:manager) { FactoryGirl.create(:ext_management_system) }
+    let(:catalog_item_options) do
+      {
+        :name         => 'Orchestration Template',
+        :service_type => 'atomic',
+        :prov_type    => 'generic_orchestration',
+        :display      => 'false',
+        :description  => 'a description',
+        :config_info  => {
+          :template_id => template.id,
+          :manager_id  => manager.id,
+          :provision   => {
+            :fqname            => ra1.fqname,
+            :service_dialog_id => service_dialog.id
```

Comment:
@jntullo Let's change `:service_dialog_id` to `:dialog_id`. This is inline with the column name for `ResourceAction` You need to change in the base method too. Thanks.